security/src/main/java/com/networknt/security/JwtVerifier.java

```diff
@@ -214,19 +223,19 @@ public JwtClaims verifyJwt(String jwt, boolean ignoreExpiry, boolean isToken) th
      * token in the request header to renew the expired token.
      *
      * @param jwt String of Json web token
-     * @param ignoreExpiry If true, don't verify if the token is expired.
+     * //@param ignoreExpiry If true, don't verify if the token is expired.
      * @param isToken True if the jwt is an OAuth 2.0 access token
      * @param getKeyResolver How to get VerificationKeyResolver
      * @return JwtClaims object
      * @throws InvalidJwtException InvalidJwtException
      * @throws ExpiredTokenException ExpiredTokenException
      */
-    public JwtClaims verifyJwt(String jwt, boolean ignoreExpiry, boolean isToken, BiFunction<String, Boolean, VerificationKeyResolver> getKeyResolver)
+    public JwtClaims verifyJwt(String jwt, boolean isToken, BiFunction<String, Boolean, VerificationKeyResolver> getKeyResolver)
```

Comment:
Will this break backwards compatability (since you are changing a method signature)? You did so as well in line 214